ssl/statem/statem_clnt.c

```diff
@@ -235,7 +235,7 @@ int ossl_statem_client_read_transition(SSL_CONNECTION *s, int mt)
      * Note that after writing the first ClientHello we don't know what version
      * we are going to negotiate yet, so we don't take this branch until later.
      */
-    if (SSL_CONNECTION_IS_TLS13(s)) {
+    if (SSL_CONNECTION_IS_TLS13(s) || SSL_CONNECTION_IS_DTLS13(s)) {
```

Comment:
I wonder if adding SSL_CONNECTION_IS_VERSION13() macro would be worth it. IMO it would improve readability of this code.

Reply:
Yeah I had the same thought. I'll do it!